src/Makefile.qt.include

```diff
@@ -268,7 +268,8 @@ RES_ICONS = \
   qt/res/icons/tx_output.png \
   qt/res/icons/tx_mined.png \
   qt/res/icons/warning.png \
-  qt/res/icons/verify.png
+  qt/res/icons/verify.png \
+  qt/res/icons/transaction_abandoned.png
```

Comment:
It's already optimized and the source SVG is not required because it's part of Hutchings typicons (we only added sourced for manipulated or self-created icons).
Also `src/qt/res/icons/transaction*.png` covers the copyright info in `contrib/debian/copyright`.